Linking.md

```diff
@@ -228,21 +253,22 @@ and where a `ComdatSym` is encoded as:
 |          |                | and must not reference an import.           |
 
 
-Merging Global Section
-----------------------
+Merging Wasm Globals
+--------------------
+
+Merging of the Wasm globals requires the re-numbering of globals.  This
+follows the normal rules for defining symbols: if two objects provide the
+same global symbol strongly, there is a link error, and so on.
 
-Global data symbols (C/C+ globals) are represented in the object file as wasm
-globals.  Defined symbols are modeled as exported I32 globals that contain the
-address of the symbol in linear memory.  Undefined globals are modeled as
-imported I32 globals.  These wasm globals are not used at runtime (i.e. there
-are no `get_global/set_global` instructions that reference them) but are instead
-referenced by `R_WEBASSEMBLY_MEMORY_ADDR*` relocation entries.
+The only global currently used is the `__stack_pointer` global, provided
+weakly by all object files, and not exported.  The final link stage may or
+may not provide an export for it, depending on the user choice, or may
+discard the definition provided by the object files and turn it into an
+import.  The "mutable globals" proposal will determine what can be done...
 
-In the final linked binary all these globals are resolved and the only remaining
-wasm global is the one that stores the explicit stack pointer.
 
-Merging Function Sections
--------------------------
+Merging Wasm Functions
```

Comment:
There is only one type of function so I'm not sure we need to rename this section.

Reply:
I was thinking in distinction to symbolic functions / function symbols. You're right it's unclear